ilp-core/src/main/java/org/interledger/core/InterledgerFulfillment.java

```diff
@@ -152,18 +156,18 @@ public int compareTo(InterledgerFulfillment other) {
         return 0;
       }
 
-      //Only call getBytes() if we have to (avoid array copy)
+      //Only call getHash() if we have to (avoid array copy)
```

Comment:
//Only call getHash()

 //Only call getPreimage()

Reply:
This is fixed.